doc/contributing.mdwn

```diff
@@ -0,0 +1,200 @@
+the Django enforcement manual.
+
+> Note: We have so far determined that writing an explicit enforcement
+> policy is not necessary, considering the available literature
+> already available online and the relatively small size of the
+> community. This may change in the future if the community grows
+> larger.
+
+# Patches
+
+Patches can be submitted through [pull requests][] on the
+[Github project][]. Some guidelines for patches:
+
+[pull requests]: https://github.com/linkcheck/linkchecker/pulls
+[Github project]: https://github.com/linkcheck/linkchecker
+
+Some guidelines for patches:
+
+* A patch should be a minimal and accurate answer to exactly one
+  identified and agreed problem.
+* A patch must compile cleanly and pass project self-tests on at least
+  the principle target platform.
+* A patch commit message must consist of a single short (less than 50
+  characters) line stating the a summary of the change, followed by a
+  blank line and then a description of the problem being solved and
+  its solution, or a reason for the change. Write more information,
+  not less, in the commit log.
+
+Patches should be reviewed by at least one other maintainer.
+
+Maintainers should not merge their own patches unless there is no
+response from other maintainers within a reasonable time frame (1-2
+days).
+
+> Note: Those guidelines were inspired by the
+> [Collective Code Construct Contract][C4]. The document was found to
+> be a little too complex and hard to read and wasn't adopted in its
+> entirety. See this [discussion][] for more information.
+
+ [C4]: https://rfc.zeromq.org/spec:42/C4/
+ [discussion]: https://github.com/zeromq/rfc/issues?utf8=%E2%9C%93&q=author%3Aanarcat%20
+
+## Patch triage
+
+You can also review existing pull requests, by cloning the
+contributor's repository and testing it. If the tests do not pass
+(either locally or in Travis), if the patch is incomplete or otherwise
+does not respect the above guidelines, submit a review with "changes
+requested" with reasoning.
+
+# Issues and bug reports
+
+We want you to report issuess you find in the software. It is a
+recognized and important part of contributing to this project. All
+issues will be read and replied to politely and
+professionnally. Issues and bug reports should be filed on the
+[Github issue tracker][issues].
+
+## Issue triage
+
+Issue triage is a useful contribution as well. You can review the
+[issues][] in the Github project and, for each issue:
+
+-  try to reproduce the issue, if it is not reproducible, label it with
+   `help-wanted` and explain the steps taken to reproduce
+-  if information is missing, label it with `invalid` and request
+   specific information
+-  if the feature request is not within the scope of the project or
+   should be refused for other reasons, use the `wontfix` label and
+   close the issue
+-  mark feature requests with the `enhancement` label, bugs with
+   `bug`, duplicates with `duplicate` and so on...
```

Comment:
> A patch commit message must consist of a single short (less than 50  characters) line stating the a summary of the change, followed by a blank line and then a description of the problem being solved and its solution, or a reason for the change. Write more information, not less, in the commit log.

1. 'the a' -> a
 2. The 'commit log' is /doc/changelog.txt, in linkchecker. Maybe worth mentioning explicitly as it's not immediately obvious there is one. There isn't one in linkchecker-gui, needs creating.